nilearn/plotting/glass_brain.py

```diff
@@ -87,12 +87,16 @@ def _get_json_and_transform(direction):
     """
     direction_to_view_name = {'x': 'side',
                               'y': 'front',
-                              'z': 'top'}
+                              'z': 'top',
+                              'l': 'left_side',
+                              'r': 'side'}
```

Comment:
r reuses the same image as x. So I left it with the same id when getting info from the files.